packages/suite/src/utils/suite/device.ts

```diff
@@ -135,12 +135,15 @@ export const isSelectedDevice = (selected?: TrezorDevice | Device, device?: Trez
 export const getFwUpdateVersion = (device: AcquiredDevice) =>
     device.firmwareRelease?.release?.version?.join('.') || null;
 
-export const getCoinUnavailabilityMessage = (reason: UnavailableCapability) => {
+export const getCoinUnavailabilityMessage = (
+    reason: UnavailableCapability,
+    deviceModel: Exclude<DeviceModel, DeviceModel.UNKNOWN>,
+) => {
     switch (reason) {
         case 'no-capability':
             return 'FW_CAPABILITY_NO_CAPABILITY';
         case 'no-support':
-            return 'FW_CAPABILITY_NO_SUPPORT';
+            return `FW_CAPABILITY_NO_SUPPORT_T${deviceModel}` as const;
```

Comment:
this would list the keys in list-unused script unless you add `dynamic: true,` in the messages or rewrite this to list all the possibilities statically (this would have the advantage that we wouldn't forget about it when replacing all to new internal names)

Reply:
Have you also discussed the option to say just "Not supported"/"Not supported by current device"...?

Reply:
I was thinking that I will maybe drop this change to avoid having three same strings and replace it 
`FW_CAPABILITY_NO_SUPPORT` which would take variable of deviceName (Trezor Model R, Trezor Model T, Trezor Model 1)

`Not supported on ${deviceName}`